service/controller/resource/workermigration/create_test.go

```diff
@@ -570,6 +606,12 @@ func TestFinishedMigration(t *testing.T) {
 	}
 	cr := o.(*providerv1alpha1.AzureConfig)
 
+	m.
```

Comment:
should we also expect `CreateOrUpdate` to be called to update the rules?

Reply:
I just read your comment about skipping tests for now. Moving on.

Reply:
Not in existing test cases since the mock returns `nil` for security groups. Therefore none of the loops ever gets executed and therefore nothing needs update.